starter-code/javascript/chronometer.js

```diff
@@ -1,10 +1,79 @@
 class Chronometer {
-  constructor() {}
-  // startClick() {}
-  // getMinutes() {}
-  // getSeconds() {}
-  // twoDigitsNumber() {}
-  // stopClick() {}
-  // resetClick() {}
-  // splitClick() {}
+
+    constructor() {
+        this.currentTime = 0;
+        this.Milisecond = 0;
+        this.miliIntervalId = 0;
+        this.intervalId = 0;
+        this.status = "stopped";
+    }
+    start() {
+
+        this.status = "running";
+
+    }
+    stop() {
+
+        this.status = "stopped";
+
+    }
+    startClick(callback) {
+
+        this.start();
+        this.miliIntervalId = setInterval(() => {
```

Comment:
This function will very soon make ur browser freeze ... u actually need a supercomputer to execute one function 1000 times in 1 sec :)